package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk-common",
-  "version": "1.4.0",
+  "version": "1.4.2",
```

Comment:
1.4.1 is for logging the allocation key

Does incrementing the patch version seem right here?

Reply:
Super subjective but I'd think the minor version number as external-facing functionality is added (vs just a bugfix). That's my loosely-held opinion; I really don't care too much either way you increment it though.